etcdctl/ctlv3/command/ep_command.go

```diff
@@ -124,14 +125,34 @@ func epHealthCommandFunc(cmd *cobra.Command, args []string) {
 			// endpoint is health.
 			ctx, cancel := commandCtx(cmd)
 			_, err = cli.Get(ctx, "health")
-			cancel()
 			eh := epHealth{Ep: ep, Health: false, Took: time.Since(st).String()}
 			// permission denied is OK since proposal goes through consensus to get it
 			if err == nil || err == rpctypes.ErrPermissionDenied {
 				eh.Health = true
 			} else {
 				eh.Error = err.Error()
 			}
+
+			if eh.Health == true {
```

Comment:
if !eh.Health ?

We should check the unhealthy case rather than the healthy case, right?